### PR TITLE
Add param to refer to custom repo

### PR DESCRIPTION
**Pull request by SaikumarRagala**
## Description
</br>
Added the param RepoName to gads for refering to custom repo name

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/SVT/SVT.ps1 [hunk 1]
```diff
@@ -249,7 +249,13 @@ function Get-AzSKADOSecurityStatus
...
 		[Parameter(HelpMessage="Skip organization and user controls.")]
 		[Alias("souc")]
-		$SkipOrgUserControls
+		$SkipOrgUserControls,
+
+		[string]
+		[Parameter(Mandatory = $false, HelpMessage="Name of the repo to fetch org policy.")]
+		[ValidateNotNullOrEmpty()]
+		[Alias("rp")]
+		$RepoName
```
Comment (abhaydaga) [inline src/AzSK.ADO/SVT/SVT.ps1:258]: use PolicyRepoName as it can overload with Repo SVT in future

File: src/AzSK.ADO/SVT/SVT.ps1 [hunk 2]
```diff
@@ -249,7 +249,13 @@ function Get-AzSKADOSecurityStatus
 		[switch]
 		[Parameter(HelpMessage="Skip organization and user controls.")]
 		[Alias("souc")]
-		$SkipOrgUserControls
+		$SkipOrgUserControls,
+
+		[string]
+		[Parameter(Mandatory = $false, HelpMessage="Name of the repo to fetch org policy.")]
```
Comment (abhaydaga) [inline src/AzSK.ADO/SVT/SVT.ps1:255]: Name of the repository containing org policy endpoint.

File: src/AzSK.ADO/SVT/SVT.ps1 [hunk 3]
```diff
@@ -249,7 +249,13 @@ function Get-AzSKADOSecurityStatus
...
+		[ValidateNotNullOrEmpty()]
+		[Alias("rp")]
```
Comment (abhaydaga) [inline src/AzSK.ADO/SVT/SVT.ps1:257]: prn

File: src/AzSK.ADO/SVT/SVT.ps1 (hunk 1 above)
Comment (SaikumarRagala) [reply src/AzSK.ADO/SVT/SVT.ps1]: Changed as suggested

File: src/AzSK.ADO/SVT/SVT.ps1 (hunk 2 above)
Comment (SaikumarRagala) [reply src/AzSK.ADO/SVT/SVT.ps1]: Changed as suggested

File: src/AzSK.ADO/SVT/SVT.ps1 (hunk 3 above)
Comment (SaikumarRagala) [reply src/AzSK.ADO/SVT/SVT.ps1]: Changed as suggested

**Review by abhaydaga — APPROVED** [review]

**Pull request merged**